app/MMB/src/scenes/timeline/events/LeaveForAirportTimelineEvent.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import * as React from 'react';
+import { Translation, TranslationFragment } from '@kiwicom/mobile-localization';
+import { Text, Color, StyleSheet } from '@kiwicom/mobile-shared';
+
+import type { TimelineEvent as TimelineEventType } from '../__generated__/TimelineQuery.graphql';
+import TimelineEvent from '../TimelineEvent';
+import TimelineEventIcon from '../TimelineEventIcon';
+
+type Props = {|
+  +data: TimelineEventType,
+|};
+
+const LeaveForAirportTimelineEvent = (props: Props) => {
+  return (
+    <TimelineEvent
+      data={props.data}
+      iconVertLines={<TimelineEventIcon name="directions-walk" />}
+      mainContent={
+        <TranslationFragment>
```

Comment:
Why do you use `TranslationFragment` here?

Reply:
I thought it was necessary if you had more than one `Translation` component in the children or further down the tree. I will replace it with `React.Fragment` then.

Reply:
It's necessary only when you require the type of the translation. That's not the case for the `mainContent` property.